Tests/DatadogTests/Datadog/Logging/LoggingMessageReceiverTests.swift

```diff
@@ -196,4 +202,96 @@ class LoggingMessageReceiverTests: XCTestCase {
         let received: LogEvent = try XCTUnwrap(core.events().last, "It should send event")
         try AssertEncodedRepresentationsEqual(received, sent)
     }
+
+    // MARK: - Web-view log
+
+    func testWhenValidWebLogEventPassed_itDecoratesAndPassesToWriter() throws {
+        let applicationVersion: String = .mockRandom()
+        let environment: String = .mockRandom()
+        let mockSessionID: UUID = .mockRandom()
+
+        let core = PassthroughCoreMock(
+            context: .mockWith(
+                env: environment,
+                version: applicationVersion,
+                serverTimeOffset: 123,
+                featuresAttributes: [
+                    "rum": [
+                        RUMContextAttributes.applicationID: "123456",
+                        RUMContextAttributes.sessionID: mockSessionID.uuidString.lowercased()
+                    ]
+                ]
+            ),
+            messageReceiver: LoggingMessageReceiver(logEventMapper: nil)
+        )
+
+        let webLogEvent: JSON = [
+            "date": 1_635_932_927_012,
+            "error": ["origin": "console"],
+            "message": "console error: error",
+            "session_id": "0110cab4-7471-480e-aa4e-7ce039ced355",
+            "status": "error",
+            "view": ["referrer": "", "url": "https://datadoghq.dev/browser-sdk-test-playground"]
+        ]
+
+        let expectedWebLogEvent: JSON = [
+            "date": 1_635_932_927_012 + 123.toInt64Milliseconds,
+            "error": ["origin": "console"],
+            "message": "console error: error",
+            "application_id": "123456",
+            "session_id": mockSessionID.uuidString.lowercased(),
+            "status": "error",
+            "ddtags": "version:\(applicationVersion),env:\(environment)",
+            "view": ["referrer": "", "url": "https://datadoghq.dev/browser-sdk-test-playground"]
+        ]
+
+        core.send(
+            message: .custom(
+                key: LoggingMessageKeys.browserLog,
+                baggage: .init(webLogEvent)
+            )
+        )
+
+        let data = try JSONEncoder().encode(core.events.first as? AnyEncodable)
+        let writtenJSON = try XCTUnwrap(try JSONSerialization.jsonObject(with: data, options: []) as? JSON)
+
+        AssertDictionariesEqual(writtenJSON, expectedWebLogEvent)
```

Comment:
nit/ Wouldn't this be more convenient?
```suggestion
        let received = try XCTUnwrap(core.events.first, "It should send event")
        try AssertEncodedRepresentationsEqual(received, AnyEncodable(expectedWebLogEvent))
```

Reply:
Yes indeed 👍